0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -175,6 +180,31 @@ ProtocolData ::= SEQUENCE OF SEQUENCE {
 }
 ```
 
+## Authentication
+
+Before anything else, when a client connects to a server, it sends a special
```

Comment:
It should say that if the client sends any other packet than the auth data first, the server MUST reply with an `Error` and close the connection.